projects/core/src/user/connectors/replenishment-order/user-replenishment-order.connector.spec.ts

```diff
@@ -129,3 +134,39 @@ describe('UserReplenishmentOrderConnector', () => {
     );
   });
 });
+
+describe('UserReplenishmentOrderConnector', () => {
+  let adapter: UserReplenishmentOrderAdapter;
```

Comment:
please remove and just add the `it` function assertion to above existing `describe` function